src/Operations.jl

```diff
@@ -1166,6 +1166,7 @@ function rm(ctx::Context, pkgs::Vector{PackageSpec})
     end
     deps_names = collect(keys(ctx.env.project["deps"]))
     if haskey(ctx.env.project, "targets")
+        append!(deps_names, keys(ctx.env.project["extras"]))
```

Comment:
@00vareladavid (or anyone really) 27d452d73ed33ac368b3cbf6675a2eaca1ff39a9 did not backport cleanly, does this look like an OK backport?

Reply:
Yeah, this seems OK. Doesn't it need a `collect` in there though?